File: MyApplication2/app/src/main/java/com/example/franco/myapplication2/utilities/NetworkUtils.java
package com.example.franco.myapplication2.utilities;

import android.net.Uri;
import android.util.Log;

import java.io.IOException;
import java.io.InputStream;
import java.net.HttpURLConnection;
import java.net.MalformedURLException;
import java.net.URL;
import java.util.Scanner;

/**
 * Created by franco on 9/27/17.
 */

public class NetworkUtils {
    final static  String ID_CANCHA = "59b0ac5ab729bb0042bfcc8b";
    final static String CANCHAS_URL =
            "https://canchas.ml/clubes";

    public static String getResponseFromHttpUrl() throws IOException {
        URL url = new URL(CANCHAS_URL + '/' + ID_CANCHA);
        HttpURLConnection urlConnection = (HttpURLConnection) url.openConnection();
        try {
            InputStream in = urlConnection.getInputStream();

            Scanner scanner = new Scanner(in);
            scanner.useDelimiter("\\A");

            boolean hasInput = scanner.hasNext();
            if (hasInput) {
                return scanner.next();
            } else {
                return null;
            }
        } finally {
            urlConnection.disconnect();
        }
    }
}

File: MyApplication3/app/src/main/java/com/example/franco/myapplication/NetworkUtils.kt
package com.example.franco.myapplication

/**
 * Created by franco on 10/9/17.
 */

import java.io.IOException
import java.net.HttpURLConnection
import java.net.URL
import java.util.Scanner

/**
 * Created by franco on 9/27/17.
 */

class NetworkUtils {

    fun responseFromHttpUrl(url: URL): String? {

        val urlConnection = url.openConnection() as HttpURLConnection
        try {
            val `in` = urlConnection.inputStream

            val scanner = Scanner(`in`)
            scanner.useDelimiter("\\A")

            val hasInput = scanner.hasNext()
            return if (hasInput) {
                scanner.next()
            } else {
                null
            }
        } finally {
            urlConnection.disconnect()
        }
    }

}